runtime/src/lib.rs

```diff
@@ -403,6 +374,8 @@ parameter_types! {
     pub const BountyUpdatePeriod: u32 = 20;
     pub const DataDepositPerByte: u64 = 1;
     pub const TreasuryPalletId: PalletId = PalletId(*b"py/trsry");
+    pub const MaxApprovals: u32 = 100;
+
 }
 
 impl pallet_treasury::Config for Runtime {
```

Comment:
could we make this similar to our standalone chain? https://github.com/DataHighway-DHX/node/pull/215/files#diff-0ec06ea58bd455f09ce6b3bb4c2c1c0d37bda51c1e1be2151c560c9c973959ecR683

Reply:
`MaxApprovals` is same as standalone chain. https://github.com/DataHighway-DHX/node/blob/luke/MMD-1309/update-substrate-3/runtime/src/lib.rs#L680